UPGRADE-3.x.md

```diff
@@ -1,6 +1,12 @@
 UPGRADE 3.x
 ===========
 
+UPGRADE FROM 3.6 to 3.6.1
```

Comment:
Please remove line 4 and 5 and use a headline like (@see line 21):
`### Providing a 2nd parameter for Sonata\MediaBundle\Metadata\ProxyMetadataBuilder::__construct() is deprecated`